Dockerfile

```diff
@@ -0,0 +1,63 @@
+from ubuntu:latest
+
+# Install dependencies.
+RUN apt update -y
+RUN DEBIAN_FRONTEND="noninteractive" apt install -y tzdata
+RUN apt install -y \
+	git-all \
+	vim \
+	nano \
+	whois \
+	openssh-server \
+	curl \
+	apt-utils \
+	iputils-ping \
+	zsh \
+	tmux
+
+# Create the required users. The game master is the `git` account, and the player is the user's account
+# TODO - change the gamemaster password?
+# TODO - change the gamemaster username to git?
+RUN useradd --comment "GameMaster account" --create-home --password $(mkpasswd -m sha-512 gamemaster) gamemaster
+RUN useradd --comment "Player account" --create-home --password $(mkpasswd -m sha-512 player) --shell /bin/zsh player
```

Comment:
Isn't it /usr/bin/zsh?
Anyways, probably better to use $(which zsh)

Reply:
Fixed